lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
```

Comment:
Like most elements, the form should probably have some default spacing from whatever comes before and after it, vertically

Reply:
> Like most elements, the form should probably have some default spacing from whatever comes before and after it, vertically

Spoke with @dturner32 and he would like 32px of bottom margin: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/c440080d4bcc0749d3df053adf38dbb4e0068cdc